src/o365/spo/commands/web/web-add.ts

```diff
@@ -0,0 +1,313 @@
+    telemetryProps.webTemplate = (!(!args.options.webTemplate)).toString();
+    telemetryProps.parentWebUrl = (!(!args.options.parentWebUrl)).toString();
+    telemetryProps.description = (!(!args.options.description)).toString();
+    telemetryProps.locale = (!(!args.options.locale)).toString();
+    telemetryProps.breakInheritance = args.options.breakInheritance || false;
+    telemetryProps.inheritNavigation = args.options.inheritNavigation || false;
+
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const resource: string = Auth.getResourceFromUrl(args.options.parentWebUrl);
+    let siteAccessToken: string = '';
+    
+    auth
+    .ensureAccessToken(resource, cmd, this.debug)
+    .then((accessToken: string): Promise<boolean> => {
+      if (this.debug) {
+        cmd.log(`Retrieved access token ${accessToken}. Retrieving request digest...`);
+      }
+      siteAccessToken = accessToken;
+      return this.createWeb(accessToken, cmd, args, cb, this.debug);
+    })
+    .then((res: boolean) : any => {
+      if(args.options.inheritNavigation)
+      {
+        if(this.debug)
+        {
+          cmd.log("Setting the navigation as the parent site.");
+        }
+        this.getEffectiveBasePermission(siteAccessToken, cmd, args, cb, this.debug)
+        .then((perm:BasePermissions) : any => {
+            /// Detects if the site in question has no script enabled or not. 
+            /// Detection is done by verifying if the AddAndCustomizePages permission is missing.
+            /// 
+            /// See https://support.office.com/en-us/article/Turn-scripting-capabilities-on-or-off-1f2c515f-5d7e-448a-9fd7-835da935584f
+            /// for the effects of NoScript
+            /// 
+            if(perm.has(PermissionKind.AddAndCustomizePages)) {
+              return this.setInheritNavigation(siteAccessToken, cmd, args, cb, this.debug).then((res : any) => {
+                cb();
+              },(err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+            }
+            else {
+              cmd.log("No script is enabled. Skipping the InheritParentNavigation settings.")
+              cb();
+            }
+          }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+      }
+      else
+       {
+         cb();
+       }
+    }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-t, --title <title>',
+        description: 'Subsite title'
+      },
+      {
+        option: '-d, --description [description]',
+        description: 'Subsite description, optional'
+      },
+      {
+        option: '-u, --webUrl [webUrl]',
+        description: 'Subsite relative url'
+      },
+      {
+        option: '-w, --webTemplate [webTemplate]',
```

Comment:
webTemplate is required so should be denoted using angle brackets, like <webTemplate>